app/models/point_request.rb

```diff
@@ -11,7 +11,7 @@ class PointRequest < ActiveRecord::Base
   has_many :point_details
 
   def recalculate!
-    self.point = point_details.sum(:point)
+    self.point = ApplicationController.helpers.sum_all_details(point_details, user)
```

Comment:
I think this this is not really a good idea. I mean call controller method in modell. 
But currently, i don't know what would be better. @salierri ?

Reply:
Yes, it should be refactored as I described in the commit message, but now I don't have a better idea. Maybe it could be moved to a new view model, that could be used by the evaluation table too, but I think it's enough large change to do it in a new pull request.